### ISSUE TITLE
NVDA reading page before CDTS scripts have completed

### ISSUE THREAD

**Issue by sgdowney** [wet-boew/cdts-sgdc#1420]
cdts-WebTemplates version: 
GoC.WebTemplate Version MVC 5.0.9264.32561
GoC.WebTemplate.Core Version 5.0.9264.32634
Template: Application Template
CDTS Version: v5_0_4

Using NVDA, it starts reading out the page before the CDTS scripts have updated the DOM.  The **Switch to Basic HTML** link is not announced. The text in the web application name link is not updated yet so **Name of Web Application** is announced. Manually tabbing through the page after it has loaded does properly announce the above mentioned items.

Is there a way to prevent NVDA (and probably other screen readers) from reading the page until the scripts have finished updating the DOM?
